patches/minecraft/net/minecraft/world/entity/monster/Shulker.java.patch

```diff
@@ -0,0 +1,15 @@
+--- a/net/minecraft/world/entity/monster/Shulker.java
++++ b/net/minecraft/world/entity/monster/Shulker.java
+@@ -356,6 +_,12 @@
+             if (blockpos1.m_123342_() > this.f_19853_.m_141937_() && this.f_19853_.m_46859_(blockpos1) && this.f_19853_.m_6857_().m_61937_(blockpos1) && this.f_19853_.m_45756_(this, (new AABB(blockpos1)).m_82406_(1.0E-6D))) {
+                Direction direction = this.m_149810_(blockpos1);
+                if (direction != null) {
++                  net.minecraftforge.event.entity.EntityTeleportEvent.EnderEntity.EnderEntity event = net.minecraftforge.event.ForgeEventFactory.onEnderTeleport(this, blockpos1.m_123341_(), blockpos1.m_123342_(), blockpos1.m_123343_());
```

Comment:
`net.minecraftforge.event.entity.EntityTeleportEvent.EnderEntity.EnderEntity` does not look correct

Reply:
Whoops, my copy-paste of the class name must've been faulty. Fixed.